src/aap_eda/core/enums.py

```diff
@@ -87,3 +87,17 @@ class CredentialType(DjangoStrEnum):
     REGISTRY = "Container Registry"
     GITHUB = "GitHub Personal Access Token"
     GITLAB = "GitLab Personal Access Token"
+
+
+ACTIVATION_STATUS_MESSAGE_MAP = {
+    ActivationStatus.PENDING: "Wait for worker to start the activation",
```

Comment:
This would not be accurate in all cases. Pending is our default initial status. when we create a disabled activation is this its status. We should have a "disabled" status.

Reply:
@Alex-Izquierdo how about based on `is_enabled` flag, we set different default message? 
```
if is_enabled:
    status_message = "Wait for a worker to be available to start activation"
else:
   status_message = "Activation is marked as disabled"
```
In this case, we don't need to add `disabled` status

Reply:
I feel a pending state for an activation that is not enabled is confusing and a bad UX. But that is another discussion. For the scope of the PR, that conditional is what we need to have useful messages, IMO.